app/experimenter/experiments/forms.py

```diff
@@ -1271,10 +1272,42 @@ class NormandyIdForm(ChangeLogMixin, forms.ModelForm):
     normandy_id = forms.IntegerField(
         label="Recipe ID",
         widget=forms.TextInput(
-            attrs={"class": "form-control", "placeholder": "Recipe ID"}
+            attrs={"class": "form-control", "placeholder": "Main Recipe ID"}
         ),
     )
 
+    other_normandy_ids = forms.CharField(
+        label="Other Recipe IDs",
+        widget=forms.TextInput(
+            attrs={
+                "class": "form-control",
+                "placeholder": "Other Recipe IDs (Optional)",
+            }
+        ),
+        required=False,
+    )
+
+    def clean_other_normandy_ids(self):
+        other_ids_list = []
+
+        if self.cleaned_data["other_normandy_ids"]:
+            ids = re.split(",", self.cleaned_data["other_normandy_ids"])
+            for id in ids:
+                if re.match(r"^[0-9]+$", id.strip()):
+                    if "normandy_id" in self.cleaned_data:
+                        if int(id) == self.cleaned_data["normandy_id"]:
```

Comment:
In Django, if you want to validate multiple fields together you have to do it in the [clean method](https://docs.djangoproject.com/en/2.2/ref/forms/validation/#cleaning-and-validating-fields-that-depend-on-each-other)

Reply:
but 👍 for doing this check :D

Reply:
hmm.. so you are saying this check should be moved to the clean method?

Reply:
moved to clean method solved a few probs...